Description: Fix -mm .RD macro to use .if rather than .ie
Author: James Avera
Bug-Ubuntu: https://bugs.launchpad.net/bugs/738169
Forwarded: yes
Applied-Upstream: http://bazaar.launchpad.net/~vcs-imports/groff/main/revision/1992
Last-Update: 2011-03-19

Index: b/contrib/mm/m.tmac
===================================================================
--- a/contrib/mm/m.tmac
+++ b/contrib/mm/m.tmac
@@ -689,7 +689,7 @@
 .rd \\$1\t
 .br
 .di
-.ie !''\\$3' \{\
+.if !''\\$3' \{\
 .	di misc*rd2
 .	ds \\$3 "\\*[misc*rd]
 .	br
